libnavigation-core/src/test/java/com/mapbox/navigation/core/history/MapboxHistoryRecorderTest.kt

```diff
@@ -18,12 +24,25 @@ class MapboxHistoryRecorderTest {
 
     private val context: Context = ApplicationProvider.getApplicationContext()
     private val navigationOptionsBuilder = NavigationOptions.Builder(context)
-    private val logger: Logger = mockk(relaxUnitFun = true)
+
+    @Before
+    fun setUp() {
+        mockkStatic(Logger::class)
```

Comment:
let's move the mock logger to a separate class in the test utils module

Reply:
maybe logger rule instead of base logger?

Reply:
I like it 👍